clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -3369,9 +3369,13 @@ bool clang::format::UnwrappedLineParser::parseRequires() {
     case tok::kw_volatile:
     case tok::kw_const:
     case tok::comma:
-      FormatTok = Tokens->setPosition(StoredPosition);
-      parseRequiresExpression(RequiresToken);
-      return false;
+      if (OpenAngles == 0) {
+        FormatTok = Tokens->setPosition(StoredPosition);
+        parseRequiresExpression(RequiresToken);
+        return false;
+      } else {
+        break;
+      }
```

Comment:
I realized after that i could do this. Would this be preferred?
```suggestion
      if (OpenAngles != 0)
        break;
      FormatTok = Tokens->setPosition(StoredPosition);
      parseRequiresExpression(RequiresToken);
      return false;
```

Reply:
I prefer the latter.

Reply:
The former is more consistent with the rest of the `switch` statement, except that there should be no `else` after the `return`. So I would prefer:
```
      if (OpenAngles == 0) {
        FormatTok = Tokens->setPosition(StoredPosition);
        parseRequiresExpression(RequiresToken);
        return false;
      }
      break;
```

Reply:
> The former is more consistent with the rest of the `switch` statement, except that there should be no `else` after the `return`. So I would prefer:
> 
> ```
>       if (OpenAngles == 0) {
>         FormatTok = Tokens->setPosition(StoredPosition);
>         parseRequiresExpression(RequiresToken);
>         return false;
>       }
>       break;
> ```

Also good, no `else` is my main point.